website/content/api-docs/secret/pki.mdx

```diff
@@ -106,6 +113,321 @@ to mix different types of CAs (roots and intermediates).
+        "created_on": "2023-05-24T14:33:00-04:00",
+        "key_bits": "256",
+        "key_type": "hs"
+      }
+    },
+    "keys": [
+      "bc8088d9-3816-5177-ae8e-d8393265f7dd"
+    ]
+  }
+}
+```
+
+### Delete Unused ACME EAB Binding Tokens
+
+This endpoint allows the deletion of an unused ACME binding token.
+
+| Method   | Path                    |
+| :------- | :---------------------- |
+| `DELETE` | `/pki/acme/eab/:key_id` |
+
+#### Parameters
+
+ - `key_id` `(string: <required>)` - The id of the EAB binding token to
+   delete. This is part of the request URL.
+
+#### Sample Request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request DELETE \
+    http://127.0.0.1:8200/v1/pki/acme/eab/bc8088d9-3816-5177-ae8e-d8393265f7dd
+```
+
+### Get ACME Configuration
+
+This endpoint allows reading of the current ACME server configuration used by
+this mount.
+
+| Method | Path               |
+| :----- | :----------------- |
+| `GET`  | `/pki/config/acme` |
+
+#### Sample Request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    http://127.0.0.1:8200/v1/pki/config/acme
+```
+
+#### Sample Response
+
+```
+{
+  "data": {
+    "allowed_issuers": [
+      "*"
+    ],
+    "allowed_roles": [
+      "*"
+    ],
+    "default_role": "",
+    "dns_resolver": "",
+    "eab_policy": "not-required",
+    "enabled": true
+  },
+}
+```
+
+### Set ACME Configuration
+
+This endpoint allows setting the ACME server configuration used by this
+mount.
+
+| Method | Path               |
+| :----- | :----------------- |
+| `POST` | `/pki/config/acme` |
+
+#### Parameters
+
+ - `allowed_issuers` `(list: ["*"])` - Specifies a list issuers to allow to
+   issue certificates via explicit ACME paths. If an allowed role specifies
+   an issuer outside this list, it will be allowed. The default value `*`
+   allows every issuer within the mount.
+
+ - `allowed_roles` `(list: ["*"])` - Specifies a list of roles to allow to
+   issue certificates via explicit ACME paths. If no `default_role` is
+   specified, sign-verbatim-like issuance on the default ACME directory
+   will still occur.
+
+ - `default_role` `(string: "")` - Optionally specifies a role to enforce
+   on the default ACME directory. Must be present in `allowed_roles` if
+   set.
+
+ - `dns_resolver` `(string: "")` - An optional overriding DNS resolver to
+   use for challenge verification lookups. When not specified, the default
+   system resolver will be used. This allows domains on peered networks with
+   an accessible DNS resolver to be validated.
+
+ - `eab_policy` `(string: "not-required")` - Specified policy to enforce
+   around [External Account Bindings (EABs)](#acme-external-account-bindings).
+   The allowed values are:
+
+     - `not-required`, where EABs are not enforced but are validated if
+        specified,
+
+     - `new-account-required`, where new accounts are required to have EAB
+       but existing accounts can still be used, and
```

Comment:
Sentence wasn't completed?

Reply:
Nah, it was just a continued list. `item one, / item-two, and / item-three.` :-)